api/tests/certification/session/unit/application/invigilator-kit-controller_test.js

```diff
@@ -10,33 +12,28 @@ describe('Unit | Controller | invigilator-kit-controller', function () {
       const sessionMainInfo = domainBuilder.buildSessionForInvigilatorKit({ id: 1 });
       const invigilatorKitBuffer = 'binary string';
       const userId = 1;
+      const i18n = getI18n();
       const request = {
+        i18n,
         auth: { credentials: { userId } },
         params: { id: sessionMainInfo.id },
-        query: {
-          accessToken: 'ACCESS_TOKEN',
-        },
       };
-      const tokenService = {
-        extractUserId: sinon.stub(),
-      };
-      const requestResponseUtils = {
-        extractLocaleFromRequest: sinon.stub(),
-      };
-      tokenService.extractUserId.withArgs(request.query.accessToken).returns(userId);
+      i18n.setLocale(LANG.ENGLISH);
+
       const invigilatorKitPdf = {
         getInvigilatorKitPdfBuffer: sinon.stub(),
       };
-      invigilatorKitPdf.getInvigilatorKitPdfBuffer.resolves({
-        buffer: invigilatorKitBuffer,
-        fileName: `kit-surveillant-${sessionMainInfo.id}.pdf`,
-      });
+
       usecases.getInvigilatorKitSessionInfo.resolves(sessionMainInfo);
+      invigilatorKitPdf.getInvigilatorKitPdfBuffer
+        .withArgs({ sessionForInvigilatorKit: sessionMainInfo, lang: LANG.ENGLISH })
+        .resolves({
+          buffer: invigilatorKitBuffer,
+          fileName: `kit-surveillant-${sessionMainInfo.id}.pdf`,
```

Comment:
Comme le test expect un fichier avec un nom en français, je propose d'opter pour FRENCH plutôt que ENGLISH

```suggestion
      invigilatorKitPdf.getInvigilatorKitPdfBuffer
        .withArgs({ sessionForInvigilatorKit: sessionMainInfo, lang: LANG.FRENCH })
        .resolves({
          buffer: invigilatorKitBuffer,
          fileName: `kit-surveillant-${sessionMainInfo.id}.pdf`,
```

Reply:
Ajout d'un test pour faire verifier que les langues sont prises en compte (en + fr)